notebooks/official/automl/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -0,0 +1,1689 @@
+{
```

Comment:
<p>Can we add something to the title of this section to clarify which notebook environment these steps are for?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>The title of the section already says it is for running the notebook locally. Also, this comes from the template notebook, so we shouldn't change too much here for the sake of consistency. Instead, we should change the template and they make all existing notebooks reflect that change.</p>

Reply:
<p>Yes, this is a change I had originally proposed to the template notebook. Are you just including sections of it here, without duplicating the text?</p>

Reply:
<p>Yes, I'm following the template.</p>